my_list.rb

```diff
@@ -0,0 +1,13 @@
+require './my_enumerable'
```

Comment:
- [ ] Kindy use require relative to import your `my_enumerable`, this is ruby convection to use require relative for files in your directory and require for external files like gems.